docs/source/torch.rst

```diff
@@ -1,7 +1,7 @@
 torch
 =====
 The torch package contains data structures for multi-dimensional
-tensors and mathematical operations over these are defined.
+tensors. It also defines mathematical operations that can be performed over these tensors.
```

Comment:
Can you please change it to match torch/__init__.py? https://github.com/pytorch/pytorch/commit/96aaa311c0251d24decb9dc5da4957b7c590af6f